src/App.js

```diff
@@ -1,86 +1,84 @@
   // true = single view?
   return (
     <div className="App">
       <header className="App-header">
         <div>
           {
-            <button onClick={switchInput}> {
+            <button className="float-switch-button my-button" onClick={switchInput}> {
               isSingleView ? "View List" : "View Single"
             } </button>
           }
-          <h2> Add a New Food</h2>
-          <input type="text" onChange={(event) => { setInputFood(event.target.value) }} value={inputFood} />
-          <div>
-            <button onClick={addFood}>  Submit </button>
-          </div>
-        </div>
-        <div>
-          {isSingleView && (
+          <div className="grid-container">
             <div>
-              <Food
-                name={foodList[currIndex]}
-                index={currIndex}
-                foodList={foodList}
-                setFoodList={setFoodList}
-              >
-              </Food>
-              <button onClick={nextFood}> Pass Me </button>
+              <h2 id="main-title"> Add a New Food</h2>
+              <input className="input-field" type="text" onChange={(event) => { setInputFood(event.target.value) }} value={inputFood} />
+              <div>
+                <button className="my-button" onClick={addFood}>  Submit </button>
+              </div>
             </div>
-          )
-          }
-          {!isSingleView &&
-            foodList.map((value, index) => {
-              return <Food
-                name={value}
-                index={index}
-                foodList={foodList}
-                setFoodList={setFoodList}
-                likes={likes}
-                setLikes={setLikes}
-              >
-              </Food>
-            })
-          }
+
+            <div>
+              { foodList.length < 1 &&
+                <h4 className="text-colorset"> Please enter a food! </h4>
+              }
+              {isSingleView && foodList.length > 0 &&  (
+                <div className="food-item">
+                  <Food
+                    name={foodList[currIndex]}
+                    index={currIndex}
+                    foodList={foodList}
+                    setFoodList={setFoodList}
+                    likeCount={likeCount}
+                    setLikeCount={setLikeCount}
+                    setCurrIndex={setCurrIndex}
+                    isSingleView={isSingleView}
+                  >
+                  </Food>
+                </div>
+              )
+              }
+              {!isSingleView && foodList.length > 0 && 
+                foodList.map((value, index) => {
+                  return  <div className="food-item"> <Food
+                    name={value}
+                    index={index}
+                    foodList={foodList}
+                    setFoodList={setFoodList}
+                    likeCount={likeCount}
+                    setLikeCount={setLikeCount}
+                    setCurrIndex={setCurrIndex}
```

Comment:
nit: see how we have props `index` and `setCurrIndex`? it would be good to use the same suffix so that it's easy to see that they are referring to the same object!!

btw nit mean nitpicky, feel free to ignore/dismiss